docs/docs/components/reanimated-drawer-layout.mdx

```diff
@@ -0,0 +1,192 @@
+---
+id: reanimated-drawer-layout
+title: Reanimated Drawer Layout
+sidebar_label: Reanimated Drawer Layout
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+Cross-platform replacement for the React Native's [DrawerLayoutAndroid](http://reactnative.dev/docs/drawerlayoutandroid.html) component.  
+Please refer to [React Native docs](http://reactnative.dev/docs/drawerlayoutandroid.html) for the detailed usage for standard parameters.
+
+### Usage:
+
+To use it, import it in the following way:
+
+```js
+import ReanimatedDrawerLayout from 'react-native-gesture-handler/ReanimatedDrawerLayout';
+```
+
+## Properties:
+
+### `drawerType`
+
+accepts values of the `DrawerPosition` enum, defaults to `FRONT`.  
+Specifies the way the drawer will be displayed.
+
+- `FRONT` the drawer will be displayed above the content view.
+- `BACK` the drawer will be displayed below the content view, revealed by sliding away the content view.
+- `SLIDE` the drawer will appear attached to the content view, opening it slides both the drawer and the content view.
+
+| `FRONT`                                               | `BACK`                                               | `SLIDE`                                               |
+| ----------------------------------------------------- | ---------------------------------------------------- | ----------------------------------------------------- |
+| <img src={useBaseUrl('gifs/new-drawer-front.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-back.gif')} /> | <img src={useBaseUrl('gifs/new-drawer-slide.gif')} /> |
+
+### `edgeWidth`
+
+width of the invisible draggable area on the edge of the content view, which can be dragged to open the drawer.
+
+### `hideStatusBar`
+
+a boolean value, when set to `true`, drawer component will use [StatusBar API](http://reactnative.dev/docs/statusbar.html) to hide the OS status bar when the drawer is dragged or idle in the `open` position.
+
+### `statusBarAnimation`
+
+a string, possible values are: `slide`, `none` or `fade`, defaults to `slide`.
+May be used in combination with `hideStatusBar` to select the animation used for hiding the status bar.
+See [StatusBar API](http://reactnative.dev/docs/statusbar.html#statusbaranimation) docs.
+
+### `overlayColor`
+
+color (default `rgba(0, 0, 0, 0.7)`) of the background overlay on top of the content window when the drawer is `open`.
+This color animates from `0%` to `100%` opacity as drawer animates from `closed` to `open`.
+
+### `renderNavigationView`
+
+a renderer function for the drawer componenet.
+Provided with a `progress` SharedValue parameter that indicates the progress of drawer opening/closing. Equals `0` when drawer is `closed` and `1` when `open`.
+This can be used by the drawer component to animated its children while the drawer is opening or closing.
+
+### `onDrawerClose`
+
+a function which is called when the drawer is closed.
```

Comment:
What about 
```suggestion
a function which is called when the drawer has been closed.
```
?


I think it tells more about the timing of this action.

Reply:
I agree